FastSimulation/Tracking/plugins/TrackCandidateProducer.cc

```diff
@@ -85,7 +85,8 @@ TrackCandidateProducer::produce(edm::Event& e, const edm::EventSetup& es) {
 
   edm::ESHandle<Propagator>             propagator;
   es.get<TrackingComponentsRecord>().get(propagatorLabel,propagator);
-    
+  Propagator* thePropagator = propagator.product()->clone();
```

Comment:
>Sorry the code was crashing but for different reasons, not because of the propagator. I had removed some important stuff while doing some last minute changes to the code. 
>The propagator needs to be cloned as I am changing its propagation direction later.